AEPIdentity/Sources/MobileIdentities.swift

```diff
@@ -82,12 +85,13 @@ struct MobileIdentities: Codable {
         let configurationStatus = sharedStateProvider(IdentityConstants.SharedStateKeys.CONFIGURATION, event)?.status ?? .none
         let analyticsStatus = sharedStateProvider(IdentityConstants.SharedStateKeys.ANALYTICS, event)?.status ?? .none
         let audienceStatus = sharedStateProvider(IdentityConstants.SharedStateKeys.AUDIENCE, event)?.status ?? .none
-        // TODO: Target
+        let targetStatus = sharedStateProvider(IdentityConstants.SharedStateKeys.AUDIENCE, event)?.status ?? .none
```

Comment:
this is meant to be `IdentityConstants.SharedStateKeys.TARGET`, yes?

Reply:
Good catch!